Runtime/AvatarCreator/Scripts/WebRequests/AvatarAPIRequests.cs

```diff
@@ -83,7 +80,7 @@ public async Task<AvatarProperties> CreateFromTemplateAvatar(string templateId,
             var payloadData = new Dictionary<string, string>
             {
                 { nameof(partner), partner },
-                { nameof(bodyType), GetBodyTypeValue(bodyType) }
+                { nameof(bodyType), bodyType.GetDescription() }
```

Comment:
does this CreateFromTemplateAvatar function really need a BodyType parameter now? shouldn't it be coming from core settings? 🤔